src/components/UpcomingEventCard.vue

```diff
@@ -197,6 +202,10 @@ export default {
 				/(\b(https?|ftp|file):\/\/[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/gi; // eslint-disable-line
 			return text.match(urlRegex);
 		},
+		getAdminlyColor(name) {
+			const index = name.length > 12 ? name.length - 12 : name.length;
```

Comment:
What if the name is 26 characters long?

Reply:
true 😅

Reply:
@oliveiraigorm You could actually skip the entire conditional and just have index be `name.length % 12` because if it's below 12 it'll just be whatever number it is.

Reply:
Just changed the true case and didn't even stop to read the full thing🤦